reana-ui/src/registerServiceWorker.js

```diff
@@ -0,0 +1,117 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
I would remove this, we are not going to use Service Workers now, let see what happens after first implementation.

Reply:
Do you mean removing the whole file? or just that line?